src/lib/__tests__/artworkToSubmissionMapping.test.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  artworkToSubmissionCategory,
+  artworkToSubmissionMapping,
+} from "lib/artworkToSubmissionMapping"
+
+describe("artworkToSubmissionMapping", () => {
+  it("doesn't fail with empty artwork", () => {
+    const artwork = {}
+    const submission = artworkToSubmissionMapping(artwork)
+
+    expect(submission).toEqual({})
+  })
+
+  it("correctly maps artwork to submission", () => {
+    const artwork = {
+      artist: { id: "artist-id" },
+      title: "Artwork Title",
+      dates: ["2020"],
+      medium: "Medium",
+      category: "Photography",
+      attribution_class: "Attribution Class",
+      edition_sets: [{ available_editions: [1], edition_size: "2" }],
+      height: "10",
+      width: "10",
+      depth: "10",
+      metric: "in",
+      provenance: "Provenance",
+      collector_location: {
+        city: "City",
+        country: "Country",
+        state: "State",
+        countryCode: "US",
+        postalCode: "10001",
+      },
+    }
+
+    const submission = artworkToSubmissionMapping(artwork)
+
+    expect(submission).toEqual({
```

Comment:
This is what I often use in these cases to save time by not having to write down the expected values.

```suggestion
    expect(submission).toMatchInlineSnapshot()
```